cmd/judge.go

```diff
@@ -73,11 +74,11 @@ func parseExecuteFlags(cmd *cobra.Command, flags *executeFlags) (*judge.Executor
 	if err := json.Unmarshal([]byte(flags.solutionRequestHeaders), &solutionHeaders); err != nil {
 		return nil, fmt.Errorf("error unmarshaling problem solution request headers: %w", err)
 	}
-	interactorProgramService, err := services.NewProgramService(flags.interactorProgramService, nil)
+	interactorProgramService, err := program.NewProgramService(flags.interactorProgramService, nil)
```

Comment:
As we spoke, we're changing `NewProgramService` and `NewFileService` to `NewService` in both packages.